buildSrc/src/main/kotlin/GitInfo.kt

```diff
@@ -0,0 +1,14 @@
+import org.gradle.api.Project
+import java.io.ByteArrayOutputStream
+
+fun Project.runCommand(command: String): String {
+    val byteOut = ByteArrayOutputStream()
+    project.exec {
+        commandLine = listOf( "sh", "-c", command)
+        standardOutput = byteOut
+    }
+    return byteOut.toString()
+}
+
+fun Project.gitLatestTag(): String =
+        runCommand("git tag | grep -E '^[0-9]' | sort -V | tail -1").trim()
```

Comment:
Nice catch. I wanted a command that gets the highest numbered version tag, not just the most recent tag because I was originally including the `latest` publish config key in the npm package which has to always be semantically increasing. The naive pattern match here only works for the simple case, whoops!

We can live with the most recent tag instead (i.e. what `git describe` provides) now as I removed the `latest` tag from the npm package and it can publish more recent tags that refer to older versions just fine.

Git - several dozen ways to do everything and none of them are *quite* what you want. :-/